f5/bigip/resource.py

```diff
@@ -295,6 +305,37 @@ def _handle_requests_params(self, kwargs):
             requests_params.update({'params': params})
         return requests_params
 
+    def _check_exclusive_parameters(self, **kwargs):
```

Comment:
Since these are changes in resource.py, I'd like to see new unit tests to make sure the functionality is as we suspect. https://github.com/F5Networks/f5-common-python/blob/development/f5/bigip/test/test_resource.py